cloud/amazon/sqs_queue.py

```diff
@@ -113,16 +148,23 @@ def create_or_update_sqs_queue(connection, module):
     try:
         queue = connection.get_queue(queue_name)
         if queue:
-            # Update existing
+            # Update existing            
             result['changed'] = update_sqs_queue(queue, check_mode=module.check_mode, **queue_attributes)
-
         else:
             # Create new
             if not module.check_mode:
                 queue = connection.create_queue(queue_name)
                 update_sqs_queue(queue, **queue_attributes)
             result['changed'] = True
-
+        
+        if not module.check_mode:
```

Comment:
AFAIKS wouldn't it make sense to check for queue instad of module.check_mode? I mean queue maybe exists and so the results can be done as well. right? Did I miss anything?